openlibrary/macros/Pager.html

```diff
@@ -19,13 +19,13 @@
     <div class="clearfix"></div>
     <div class="pagination">
     $if page != 1:
-        <a href="$changequery(page=None)" class="ChoosePage">&laquo;&nbsp;First</a>
-        <a href="$changequery(page=page-1)" class="ChoosePage">&lt;&nbsp;Previous</a>
+        <a href="$changequery(page=None)" class="ChoosePage">&laquo;&nbsp;$_'(First')</a>
```

Comment:
Typo

Reply:
Fixed in 5d216dc0ff4fbdbd85d2f96dc64942dc22803e08